hazelcast/src/main/java/com/hazelcast/nio/tcp/AbstractIOSelector.java

```diff
@@ -95,6 +99,11 @@ public final void addTask(Runnable runnable) {
         selectorQueue.add(runnable);
     }
 
+    @Probe
+    private long idleTime() {
+        return Math.abs(System.currentTimeMillis() - lastSelectTimeMs);
```

Comment:
shouldn't this be `Math.max(System.currentTimeMillis() - lastSelectTimeMs, 0);`

as currently it can give you a false impression of staleness. 

let's assume there is a long GC pause right after the `System.currentTimeMillis()`. Then the `lastSelectTimeMs` might be greater than the value returned by `currentTimeMillis()` -> it will be reported as positive idle time which is simply not true.